schema/proto3/goalstate.proto

```diff
@@ -23,15 +23,19 @@ option java_package = "com.futurewei.alcor.schema";
 import "vpc.proto";
 import "subnet.proto";
 import "port.proto";
+import "neighbor.proto";
 import "securitygroup.proto";
 import "dhcp.proto";
+import "router.proto";
 
 message GoalState {
     uint32 format_version = 1;
 
     repeated VpcState vpc_states = 2;
     repeated SubnetState subnet_states = 3;
     repeated PortState port_states = 4;
-    repeated SecurityGroupState security_group_states = 5;
-    repeated DHCPState dhcp_states = 6;
+    repeated NeighborState neighbor_states = 5;
```

Comment:
Will replacing security_group_states with neighbor_states become a breaking change for protobuf message? I am a bit surprised that this change doesn't break our CI :-)

Reply:
I think it didn't break because we always set and get the protobuf fields by name, e.g:
  PortConfiguration_builder->set_id(port_id_4);
  PortConfiguration_builder->set_vpc_id(vpc_id_2);
  PortConfiguration_builder->set_name(port_name_4);
  PortConfiguration_builder->set_mac_address(vmac_address_4);
Therefore, it is okay even when the order of the fields changed.

Reply:
Okay, that is to say, if we merge this PR to Alcor/Master, DPM starts sending the protobuf message with updated version, existing ACA should be fine?